src/App.tsx

```diff
@@ -33,14 +34,17 @@ const App: React.FC = () => {
   }
 
   return (
-    <BrowserRouter>
-      <Suspense fallback={<Spinner color="blue" loading size={[10, 25]} type="ScaleLoader" />}>
-        <Switch>
-          <Route exact path="/login" component={Login} />
-          <Route path="/" component={HospitalRun} />
-        </Switch>
-      </Suspense>
-    </BrowserRouter>
+    <>
+      <BrowserRouter>
+        <Suspense fallback={<Spinner color="blue" loading size={[10, 25]} type="ScaleLoader" />}>
+          <Switch>
+            <Route exact path="/login" component={Login} />
+            <Route path="/" component={HospitalRun} />
+          </Switch>
+        </Suspense>
+      </BrowserRouter>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Is it a way of making this available only in "dev mode"? I am not fammiliar with react-query-devtools at all.

Reply:
> 
> 
> Is it a way of making this available only in "dev mode"? I am not fammiliar with react-query-devtools at all.

I thought so too then checked its npm page and it says:

_from the npm page React Query Devtools are not imported and used when process.env.NODE_ENV === 'production', so you don't need to worry about excluding them during a production build._